docs/doxygen-output/html/SBMLSupportFunctions_8h_source.html

```diff
@@ -93,21 +93,21 @@
 <div class="line"><a name="l00024"></a><span class="lineno">   24</span>&#160; </div>
-<div class="line"><a name="l00025"></a><span class="lineno">   25</span>&#160;<span class="keywordtype">double</span> factoriald(<span class="keywordtype">double</span> a);</div>
+<div class="line"><a name="l00025"></a><span class="lineno">   25</span>&#160;<span class="keywordtype">double</span> rr_factoriald(<span class="keywordtype">double</span> a);</div>
 <div class="line"><a name="l00026"></a><span class="lineno">   26</span>&#160; </div>
-<div class="line"><a name="l00027"></a><span class="lineno">   27</span>&#160;<span class="keywordtype">double</span> rootd(<span class="keywordtype">double</span> a, <span class="keywordtype">double</span> b);</div>
+<div class="line"><a name="l00027"></a><span class="lineno">   27</span>&#160;<span class="keywordtype">double</span> rr_rootd(<span class="keywordtype">double</span> a, <span class="keywordtype">double</span> b);</div>
 <div class="line"><a name="l00028"></a><span class="lineno">   28</span>&#160; </div>
-<div class="line"><a name="l00029"></a><span class="lineno">   29</span>&#160;<span class="keywordtype">double</span> logd(<span class="keywordtype">double</span> base, <span class="keywordtype">double</span> number);</div>
+<div class="line"><a name="l00029"></a><span class="lineno">   29</span>&#160;<span class="keywordtype">double</span> rr_logd(<span class="keywordtype">double</span> base, <span class="keywordtype">double</span> number);</div>
 <div class="line"><a name="l00030"></a><span class="lineno">   30</span>&#160; </div>
 <div class="line"><a name="l00031"></a><span class="lineno">   31</span>&#160;<span class="keywordtype">double</span> sec(<span class="keywordtype">double</span> a);</div>
 <div class="line"><a name="l00032"></a><span class="lineno">   32</span>&#160; </div>
 <div class="line"><a name="l00034"></a><span class="lineno">   34</span>&#160;<span class="keywordtype">double</span> cot(<span class="keywordtype">double</span> a);</div>
 <div class="line"><a name="l00035"></a><span class="lineno">   35</span>&#160;<span class="comment">//</span></div>
 <div class="line"><a name="l00037"></a><span class="lineno">   37</span>&#160;<span class="comment"></span><span class="keywordtype">double</span> arccot(<span class="keywordtype">double</span> a);</div>
 <div class="line"><a name="l00038"></a><span class="lineno">   38</span>&#160; </div>
-<div class="line"><a name="l00044"></a><span class="lineno">   44</span>&#160;<span class="keywordtype">double</span> arccot_negzero(<span class="keywordtype">double</span> a);</div>
+<div class="line"><a name="l00044"></a><span class="lineno">   44</span>&#160;<span class="keywordtype">double</span> rr_arccot_negzero(<span class="keywordtype">double</span> a);</div>
 <div class="line"><a name="l00045"></a><span class="lineno">   45</span>&#160; </div>
```

Comment:
Just to check here to make sure there's a reason arccot_negzero was renamed to rr_arccot_negzero, but the other functions ('cot', 'sec', 'arccot2') were not.

Reply:
The problem was that I couldn't get the LLJit to locate symbols until they exactly matched the function name - aka ones in sbmlsupport::* .

Reply:
We could make these more consistent, but would need to rename both [here](https://github.com/sys-bio/roadrunner/blob/32f589ad4ba1ed74bd9c7c6839a045cbbf04800d/source/llvm/Jit.cpp#L320) and [here](https://github.com/sys-bio/roadrunner/blob/15523ffce3d03971eecb229a2dc65b0540f36d87/source/llvm/SBMLSupportFunctions.cpp#L108).

Reply:
Ah, sounds like the naming issue is something else entirely, then.  Let's not worry about it.